store/store_test.go

```diff
@@ -0,0 +1,35 @@
+package store
+
+import "testing"
+
+func TestService_Prefix(t *testing.T) {
```

Comment:
These tests are great - really good to get more confidence on these bits of (vital) logic.